packages/firestore/src/api/database.ts

```diff
@@ -1861,11 +1872,21 @@ export class Query implements firestore.Query {
     };
   }
 
+  private validateHasExplicitOrderByForLimitToLast(query: InternalQuery): void {
```

Comment:
I was not aware there was a precedent already, otherwise I'd say the current way is better because it makes user's job easier. 

I'll raise this in chat to see how others feel about this.

Reply:
Yep, that is ok. Please do make sure that this is feasible before we expose limitToLast() publicly.